plugins/lua/hotkeys.lua

```diff
@@ -143,7 +143,7 @@ local function get_choices(hotkeys, bindings, is_inverted)
     -- adjust width of command fields so the hotkey tokens are right justified
     for _,choice in ipairs(choices) do
         local command_token = choice.text[1]
-        command_token.width = max_width - choice.hk_width - 3
+        command_token.width = max_width - choice.hk_width - 5
```

Comment:
that 5 is feeling a bit magical

Reply:
very fair, especially given the conversation we just had over TaxiService's PR : P

will fix

Reply:
done